pkg/domain/infra/tunnel/containers.go

```diff
@@ -819,6 +821,14 @@ func (ic *ContainerEngine) ContainerRun(ctx context.Context, opts entities.Conta
 	}
 
 	// Attach
+	if opts.SigProxy {
+		terminal.ProxySignalsRemote(con.ID, func(sig syscall.Signal) error {
+			killOpts := entities.KillOptions{All: false, Latest: false, Signal: strconv.FormatInt(int64(sig), 10)}
```

Comment:
sending a number in not safe, across different operating systems and architectures the numbers can vary.
We must should send the actual signal name.

I recommend to look at pkg/signal, you should add a new function there which loop over the SIgnalMap and converts the syscall.Signal into a the actual SIGNAL name.

Reply:
Done. Thanks.